src/main/java/net/mcreator/ui/init/BackgroundLoader.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2021, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.init;
+
+import net.mcreator.io.UserFolderManager;
+import net.mcreator.plugin.PluginLoader;
+import net.mcreator.themes.ThemeLoader;
+
+import java.awt.*;
+import java.io.File;
+import java.util.List;
+import java.util.*;
+import java.util.regex.Pattern;
+
+public class BackgroundLoader {
+
+	public static List<Image> loadUserBackgrounds() {
+		File[] bgfiles = UserFolderManager.getFileFromUserFolder("backgrounds").listFiles();
+		if (bgfiles != null) {
+			List<Image> images = new ArrayList<>();
+			Arrays.stream(bgfiles).forEach(f -> images.add(Toolkit.getDefaultToolkit().createImage(f.getPath())));

```

Comment:
```suggestion
			Arrays.stream(bgfiles).forEach(f -> images.add(ImageIO.read(f)));
```

Reply:
It didn't fix the problem. The PR doesn't build properly.

Reply:
Import `ImageIO`!

Reply:
Just import `ImageIO`.

Reply:
Just import `javax.imageio.ImageIO`.